nrrd/tests/test_reading.py

```diff
@@ -187,9 +194,6 @@ def test_read_simple_4d_nrrd(self):
         np.testing.assert_equal(data.dtype, np.float64)
         np.testing.assert_equal(data, np.array([[[[0.76903426]]]]))
 
-        # Test that the data read is able to be edited
-        self.assertTrue(data.flags['WRITEABLE'])
```

Comment:
Why did you remove this?

This assert has a purpose to ensure that the data is writeable. If the data is not writeable, then something is wrong with the code. I had an issue and that's why I added this.